src/graphics/lod_node.cpp

```diff
@@ -249,8 +250,44 @@ void LODNode::OnRegisterSceneNode()
     scene::ISceneNode::OnRegisterSceneNode();
 }
 
+
+void LODNode::autoComputeLevel(float scale)
+{
+    m_area *= scale;
+
+    // Amount of details based on user's input
+    float agressivity = 1.0;
+    if(UserConfigParams::m_geometry_level == 0) agressivity = 1.25;
+    if(UserConfigParams::m_geometry_level == 1) agressivity = 1.0;
+    if(UserConfigParams::m_geometry_level == 2) agressivity = 0.75;
+
+    // First we try to estimate how far away we need to draw
+    float max_draw = 0.0;
+    max_draw = sqrtf((0.5 * m_area + 10) * 200) - 10;
```

Comment:
sqrt is known to be quite slow, is it possible to avoid calling it so often?

Reply:
// First we try to ESTIMATE how far away we need to draw
It's just an estimate value. If u think it has a low performance, just change it instead of leaving an old, invalid PR.